src/main/resources/application-local.yml

```diff
@@ -4,6 +4,11 @@ notify:
 interventions-ui:
   baseurl: http://localhost:3000
 
+interventions-service:
+  baseurl: http://localhost:8080
+  locations:
+    sent-referral: "/sent-referral/{id}"
```

Comment:
i know this follows the pattern here, but i'm not sure having internal urls in config is a good idea. 

these URLs are defined in the controllers, and it would be very easy to forget to update the URL here too. is there some method we can use to get the url by controller method name or something so that it would either be correct or cause a compile time error?